src/gpuarray_buffer_cuda.c

```diff
@@ -1122,15 +1137,46 @@ static gpukernel *cuda_newkernel(gpucontext *c, unsigned int count,
       FAIL(NULL, GA_MEMORY_ERROR);
     }
 
-    ctx->err = cuModuleLoadData(&res->m, bin);
+    // for both info/err log
+    cujit_info_log = (char*)malloc(2*cujit_log_size*sizeof(char));
```

Comment:
The headers should be c++ clean, but the source files will probably not be touched by a c++ compiler.